internal/reference/reference.go

```diff
@@ -258,28 +357,36 @@ func generateFieldDescriptionsForStruct(ctx context.Context, t reflect.Type, roo
 
 			fireflyType = fmt.Sprintf("`%s`", fireflyType)
 
-			if fieldType.Kind() == reflect.Struct {
-				fieldInRootPages := false
-				fieldInSimpleTypes := false
-				for _, rootPageName := range rootPageNames {
-					if strings.ToLower(fieldType.Name()) == rootPageName {
-						fieldInRootPages = true
-						break
-					}
+			isStruct := fieldType.Kind() == reflect.Struct
+			isEnum := strings.ToLower(fieldType.Name()) == "ffenum"
+
+			fieldInRootPages := false
+			fieldInSimpleTypes := false
+			for _, rootPageName := range rootPageNames {
+				if strings.ToLower(fieldType.Name()) == rootPageName {
+					fieldInRootPages = true
+					break
 				}
-				for _, simpleTypeName := range simpleTypeNames {
-					if strings.ToLower(fieldType.Name()) == simpleTypeName {
-						fieldInSimpleTypes = true
-						break
-					}
+			}
+			for _, simpleTypeName := range simpleTypeNames {
+				if strings.ToLower(fieldType.Name()) == simpleTypeName {
+					fieldInSimpleTypes = true
+					break
 				}
+			}
 
-				link := fmt.Sprintf("#%s", strings.ToLower(fieldType.Name()))
-				if fieldInRootPages {
-					link = fmt.Sprintf("%s#%s", fieldType.Name(), strings.ToLower(fieldType.Name()))
-				} else if fieldInSimpleTypes {
-					link = fmt.Sprintf("simpletypes#%s", strings.ToLower(fieldType.Name()))
-				}
+			link := ""
+			switch {
+			case isEnum:
+				fireflyType = generateEnumList(field)
```

Comment:
Perhaps we should indicate that it is a `string` with one of these values. Right now, the way the table is rendered, it's a bit unclear that it's really just a string (depending on which enum you're looking at), because it just lists the values.

Reply:
Is this clearer?
![image](https://user-images.githubusercontent.com/6660217/168625731-f62236c8-41fd-455c-aa05-76f106e9bb66.png)

Reply:
Yeah that works, thanks